scripts/fetch_metadata.py

```diff
@@ -0,0 +1,61 @@
+# A script for fetching metadata from video file
+
+import cv2
+import argparse
+import os.path
+import json
+import logging
+
+logging.getLogger().setLevel(logging.INFO)
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-i","--input", help="input video path", required=True)
+parser.add_argument("-o","--output", help='output file path')
+args = parser.parse_args()
+
+file_path = args.input
+video_name = os.path.splitext(file_path)[0] #remove extension from file
+
+if args.output:
+    metadata_filename = args.output
+else:
+    metadata_filename = "{0}_metadata.txt".format(video_name)
```

Comment:
That's okay, but as NITPICK: you could use python3 f strings instead of old formatting
```
f"{video_name}_metadata.txt"
```